### PR TITLE
2.9.1 is out and default for docs

### PR DESCRIPTION
**Pull request by rgugliel-da**

### CODE REVIEW

File: docs/2.9.1/versions.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "daml": "2.9.1",
+  "canton": "2.9.1",
+  "daml_finance": "2.8.0",
```
Comment (rgugliel-da) [inline docs/2.9.1/versions.json:4]: This is a huge bump compared to previous 1.4.0

File: dropdown_versions [hunk 2]
```diff
@@ -1,3 +1,4 @@
+2.9.1
```
Comment (rgugliel-da) [inline dropdown_versions:1]: New

File: root [hunk 3]
```diff
@@ -1 +1 @@
-2.8.9
+2.9.1
```
Comment (rgugliel-da) [inline root:1]: Becomes the default

File: docs/2.9.1/versions.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+  "daml": "2.9.1",
+  "canton": "2.9.1",
+  "daml_finance": "1.4.1",
```
Comment (rgugliel-da) [inline docs/2.9.1/versions.json:4]: I don't understand why finance is 2.8.0 in assembly and docs is 1.4.1

Comment (yves-da) [reply docs/2.9.1/versions.json:4]: Yes, and 2.8.0 seemingly does not exist
```
Downloading daml-finance-doc-sources-2.8.0.tar.gz...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404
```

**Review by yves-da — APPROVED** [review]
The changes look good.

There seems to be a problem with finance artefact. Possibly it move somewhere else in the artefactroy?

File: docs/2.9.1/versions.json (hunk 4 above)
Comment (yves-da) [reply docs/2.9.1/versions.json]: Revert back to 1.4.1 and let the finance team follow up on it.

**Pull request merged**